frontend/src/views/camp/Admin.vue

```diff
@@ -45,6 +45,10 @@ export default {
   },
   data () {
     return {}
+  },
+  mounted () {
+    this.api.reload(this.camp())
+    this.api.reload(this.camp().materialLists)
```

Comment:
Hast du hier die Funktionsklammern vergessen? In der jetztigen Form machts glaube ich nichts.

Reply:
thx